wagn/spec/controllers/card_controller_spec.rb

```diff
@@ -322,7 +322,7 @@
         f = Card.create! type: "Cardtype", name: "Apple"
         xhr :post, :update, id: "~#{f.id}", card: {
           name: "Newt",
-          update_referencers: "false",
+          update_referers: "false",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.